.github/workflows/central-publish.yml

```diff
@@ -20,20 +20,20 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Ballerina Build
-        uses: ballerina-platform/ballerina-action/@slbeta5
+        uses: ballerina-platform/ballerina-action/@2201.0.0
         with:
           args:
-            build --skip-tests
+            build
```

Comment:
Shouldn't this be pack now?

Reply:
Removed in [59e3755](https://github.com/ballerina-platform/module-ballerinax-oracledb.driver/pull/3/commits/59e37552bf62ad713802822747419db74a5eb4d7)

Reply:
We cant remove build command altogeather. AFAIK pack does not run test cases

Reply:
Lets have the test command instead of build. Atleast as commented step